util/fipstools/delocate/delocate.go

```diff
@@ -573,8 +589,10 @@ func (d *delocation) processAarch64Instruction(statement, instruction *node32) (
 							panic("Symbol reference outside of ldr instruction")
 						}
 
-						if skipWS(parts.next) != nil || parts.up.next != nil {
-							panic("can't handle tweak or post-increment with symbol references")
```

Comment:
why was it that post-increment couldn't be handled? did you need to change something else as well to make it work?

Reply:
This was the only place where I changed to get this aspect working. It's unclear why the check was there in the first place, but it's been there since BoringSSL's first implementation of the delocate.go for ARM.
@nebeid determined that treating this offset as we normally would is safe to do, more details about that analysis can be found in the internal ARM debugging doc.

* https://boringssl-review.googlesource.com/c/boringssl/+/44848/8/util/fipstools/delocate/delocate.go